userguide/tutorials/getting_started.adoc

```diff
@@ -136,48 +136,176 @@ You can also go to http://<IP>:8080/api.html to explore the KillBill APIs.
+     --logs
+     --temp
+     --webapps
+     --work
+ 
 
-==== Tomcat configuration
 
-Recommended Tomcat configuration files can be found https://github.com/killbill/killbill-cloud/tree/master/ansible/templates/tomcat/conf[here]. The `.j2` extension is for the Jinja templating language: you must render the files first using your custom variables, which depend on your environment (take a look at an example of such variables https://github.com/killbill/killbill-cloud/blob/master/ansible/group_vars/all.yml[here]). https://ansible.sivel.net/test/[This website] can be used to perform the rendering online if you do not have Jinja installed.
+. Ensure that you have Java 8 or higher installed
 
-Once rendered, `setenv2.sh` should be placed in *CATALINA_HOME/bin* and all other files should be placed in *CATALINA_HOME/conf*.
+. Set the *JAVA_HOME* environment variable to the path of your JDK installation (For example, if JDK is installed at `C:\Software\jdk1.8.0_102`, you need to set *JAVA_HOME* to `C:\Software\jdk1.8.0_102`)
 
-Using these default configuration files, Tomcat will start on port 8080, which should be the only port open for ingress traffic in production (egress traffic should not be restricted).
+. Start Tomcat using the script *TOMCAT_HOME/bin/startup.sh* or *TOMCAT_HOME/bin/startup.bat*
+(Depending on your operating system)
 
-==== Kill Bill
+. Open a browser and type http://localhost:8080/. If Tomcat is installed properly, this should display the following:
 
-The Kill Bill configuration properties need to be added at the end of the file *CATALINA_HOME/conf/catalina.properties*. The minimal set of system properties required is the ones for the jdbc parameters (refer to our http://docs.killbill.io/latest/userguide_configuration.html[configuration guide] for more details). For example (update the values as needed):
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/getting-started/Tomcat-home.png[align=center]
 
-[source,java]
-----
-org.killbill.dao.url=jdbc:mysql://127.0.0.1:3306/killbill
-org.killbill.dao.user=killbill
-org.killbill.dao.password=killbill
-org.killbill.billing.osgi.dao.url=jdbc:mysql://127.0.0.1:3306/killbill
-org.killbill.billing.osgi.dao.user=killbill
-org.killbill.billing.osgi.dao.password=killbill
-logback.configurationFile=/var/lib/killbill/config/logback.xml
-----
 
-Place the logging configuration file https://github.com/killbill/killbill-cloud/blob/master/ansible/templates/killbill/logback.xml.j2[logback.xml] as `/var/lib/killbill/config/logback.xml` (refer to the Jinja templating language instructions above).
+==== Database Configuration
+
+As mentioned above, you need to configure the database manually. For this, you need to follow the steps given below:
+
+. Install your database (The Kill Bill core team uses *MySql*, but we also run regression tests against *MariaDB 10* and *PostgreSQL 12*. Users have also successfully deployed Kill Bill with *Oracle Percona*, *Aurora*,etc)
```

Comment:
* `MySQL` (instead of `MySql`)
* Missing space before `etc`
* I don't think `Oracle Percona` is a thing, you can just say `Percona`

Reply:
Fixed